CollectionvViewDemo/Cells/DailyEntryCollectionViewCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DailyEntryCollectionViewCell.swift
+//  CollectionvViewDemo
+//
+//  Created by Bence Pattogato on 23/08/2021.
+//
+
+import UIKit
+
+class DailyEntryCollectionViewCell: UICollectionViewCell {
```

Comment:
[nit] I saw that you use `final` on some classes, so probably would make it worth to be consistent and either use it everywhere it's possible or nowhere at all.